DEVELOPMENT.md

```diff
@@ -78,6 +78,21 @@ When enabled by default, this also works with editor plugins like
 
 [vim-fugitive]: https://github.com/tpope/vim-fugitive
 
+### iBazel: A file watcher for Bazel.
+
+Bazel is capable incrementally builds where it builds only subset of files that
```

Comment:
I'd recommend getting rid of this first sentence but if you keep it, grammar tweaks:

Bazel is capable of performing incremental builds where it builds only the subset of files that are impacted by file changes. However, it does not come with a file watcher.